ArcadiaParties.Data/Data/Seed.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ArcadiaParties.Data.Models;
+using Newtonsoft.Json;
+
+namespace ArcadiaParties.Data.Data
+{
+    public class Seed : ISeed
+    {
+        private readonly DataContext _context;
+
+        public Seed(DataContext dataContext)
+        {
+            _context = dataContext;
+        }
+
+        public async Task SeedData()
+        {
+            if (!_context.Department.Any())
```

Comment:
You just need to check whether one of tables is filled. Otherwise it can lead to an error when expected ids don't match with real.

Reply:
I've created one table check